integration-test/src/test/java/org/jboss/pnc/integration/BuildRecordSetRestTest.java

```diff
@@ -40,8 +40,7 @@
 
     public static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-    private static final String PRODUCT_REST_ENDPOINT = "/pnc-rest/rest/product/";
-    private static final String PRODUCT_VERSION_REST_ENDPOINT = "/pnc-rest/rest/product/%d/version/";
+    private static final String PRODUCT_MILESTONE_REST_ENDPOINT = "/pnc-rest/rest/product-milestone/";
```

Comment:
I'm afraid it may break UI. Am I right? 

@vibe13 @alexcreasy maybe you could take a look.